packages/key-management/test/cip8/cip30signData.test.ts

```diff
@@ -30,25 +19,23 @@ describe('cip30signData', () => {
     keyAgent = await keyAgentReady;
     asyncKeyAgent = await testAsyncKeyAgent(undefined, keyAgentReady);
     address = await asyncKeyAgent.deriveAddress(addressDerivationPath, 0);
-    witnesser = KeyManagementUtil.createBip32Ed25519Witnesser(asyncKeyAgent);
   });
 
   const signAndDecode = async (
     signWith: Cardano.PaymentAddress | Cardano.RewardAccount,
-    knownAddresses: GroupedAddress[],
-    sender?: MessageSender
+    knownAddresses: GroupedAddress[]
   ) => {
-    const dataSignature = await cip8.cip30signData({
+    const dataSignature = await cip8.cip30signData(keyAgent, {
       knownAddresses,
       payload,
-      sender,
-      signWith,
-      witnesser
+      signWith
     });
 
     const coseKey = COSEKey.from_bytes(Buffer.from(dataSignature.key, 'hex'));
     const coseSign1 = COSESign1.from_bytes(Buffer.from(dataSignature.signature, 'hex'));
 
+    console.log(dataSignature.key);
```

Comment:
These will need eslint ignores if they're intended to stay. Can they not just be assertions?

Reply:
ups, I added this for debugging purposes and forgot to remove them

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1393/commits/c132ea363e1ddf7d9fd94d0ce51aeae849d90200